core/src/main/java/org/apache/accumulo/core/clientImpl/TableOperationsImpl.java

```diff
@@ -1118,6 +1118,27 @@ public Map<String,String> modifyProperties(String tableName,
     }
   }
 
+  /**
+   * Like modifyProperties(...), but if an AccumuloException is caused by a TableNotFoundException,
+   * unwrap and rethrow the TNFE directly. This is a hacky, temporary workaround that we can use
+   * until we are able to change the public API and throw TNFE directly from all applicable methods.
+   */
+  private Map<String,String> modifyPropertiesUnwrapped(String tableName,
+      Consumer<Map<String,String>> mapMutator) throws TableNotFoundException, AccumuloException,
+      AccumuloSecurityException, IllegalArgumentException {
```

Comment:
Don't need to declare RTEs.

```suggestion
  private Map<String,String> modifyPropertiesUnwrapped(String tableName,
      Consumer<Map<String,String>> mapMutator) throws TableNotFoundException, AccumuloException,
      AccumuloSecurityException {
```